iocore/cache/I_CacheDefs.h

```diff
@@ -124,20 +124,10 @@ enum CacheFragType {
 typedef CryptoHash CacheKey;
 
 struct HttpCacheKey {
-  uint64_t
-  slice64(int i) const
-  {
-    return hash.slice64(i);
-  }
-  uint32_t
-  slice32(int i) const
-  {
-    return hash.slice32(i);
-  }
-
   int hostlen;
   const char *hostname;
   CacheKey hash;
+  CacheKey hash2;
```

Comment:
Where is this used?

Reply:
I was originally going to try to make it possible to probe for 128 bit hashes so that we could
upgrade without (effectively) clearing the cache, but this PR doesn't do that.   Removing...